cluster-up/cluster/images.sh

```diff
@@ -14,7 +14,13 @@ IMAGES[k8s-multus-1.13.3]="k8s-multus-1.13.3@sha256:c0bcf0d2e992e5b4d96a7bcbf988
 IMAGES[okd-4.1]="okd-4.1@sha256:e7e3a03bb144eb8c0be4dcd700592934856fb623d51a2b53871d69267ca51c86"
 IMAGES[okd-4.2]="okd-4.2@sha256:a830064ca7bf5c5c2f15df180f816534e669a9a038fef4919116d61eb33e84c5"
 IMAGES[okd-4.3]="okd-4.3@sha256:63abc3884002a615712dfac5f42785be864ea62006892bf8a086ccdbca8b3d38"
-IMAGES[ocp-4.3]="ocp-4.3@sha256:86a2720562809462803aa2de066a50cd725af141d4971daf04895d0b3d75d4cc"
+
+if [ -z $KUBEVIRTCI_PROVISION_CHECK ]; then
+  IMAGES[ocp-4.3]="ocp-4.3@sha256:86a2720562809462803aa2de066a50cd725af141d4971daf04895d0b3d75d4cc"
+else
+  IMAGES[ocp-4.3]="ocp-4.3-provision:latest"
```

Comment:
I don't think it's a good idea to do this change here, as it's a ocp specific thing. I'd rather put this into the ocp 4.3 `provider.sh` and work towards cleaning up that prefix oddity in a later move.

Reply:
we need it also for okd i think, so maybe worth putting logic in a common place for them ?

Reply:
done, please review